Functions.Templates/Templates/DurableFunctionsOrchestration-CSharp-Isolated/DurableFunctionsOrchestrationCSharp.cs

```diff
@@ -0,0 +1,54 @@
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.DurableTask;
+using Microsoft.Extensions.Logging;
+
+namespace Company.Function
+{
+    public static class DurableFunctionsOrchestrationCSharp
```

Comment:
This might be a dumb question, but are there any template parameters we need to consider and use for the class name? Or is it okay that we hardcode it and let developers rename it after its created?

Reply:
That is a great question haha. I think we could definitely improve this template to parameterize the name. However, given the time constraints I would prefer to leave that for a later time.

Reply:
The string `DurableFunctionsOrchestrationCSharp` is actually value of the `source` property. VS actually prompts for the functionName and replaces the source property with user input.